### DISCUSSION TITLE
Notification on main screen

### DISCUSSION THREAD

**Discussion by panason** [Blackymas/NSPanel_HA_Blueprint#1775] [Q&A] [answered]
Hello, I am using automation to call **nspanel_notification_show** with a message. After a moment, I call **nspanel_notification_clear**. On the main screen, I see the icon for the message, but when I click on it, an empty message is displayed."

**Comment by edwardtfn** [comment]
So, do you mean the notification icon still there after the notification is cleared?

Could you please report this as a bug in the issues area?

**Comment by panason** [comment]
In my opinion hiding (not deleting) notifications is the right behaviour.
Now with nspanel_notification_clear the message disappears. But there is a notification icon on the Home screen. If you tap on it, an empty message opens.

**Reply by edwardtfn** [reply]
Imthink the ic8n not disappearing is a bug, we should fix that.
About clearing vs hiding notifications, it could be another service, but if you just use the existing services to go to home page or to simulate clicking the close button you will get that result.

**Reply by edwardtfn** [reply, marked as answer]
- #1776
